tutorials/early_stopping/mnist_train_nas.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Example training code, adapted from
+https://pytorch.org/tutorials/intermediate/ax_multiobjective_nas_tutorial.html
```

Comment:
The tutorials are BDS3, so if we write this was adopted from there then technically we'd have to restate that license / its conditions here. Since we're the ones who wrote that code in the first place we don't need to, so maybe let's just remove this comment?